### PR TITLE
Test of telemetry with mqtt adapter

### PR DESCRIPTION
**Pull request by famartinrh**

### CODE REVIEW

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=shared-iot

File: systemtests/src/test/java/io/enmasse/systemtest/iot/mqtt/MqttAdapterTest.java [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                if(json.containsKey("i")) {
+                    receivedMessagesCounter.incrementAndGet();
+                    return json.getBoolean("end");
+                }
+            }
+            return false;
+        });
+
+        log.info("Sending telemetry messages");
+        for ( int i = 0; i < 50; i++ ) {
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/iot/mqtt/MqttAdapterTest.java]: Please replace magic numbers with vars

File: systemtests/src/main/java/io/enmasse/systemtest/mqtt/MqttClientFactory.java [hunk 2]
```diff
@@ -93,14 +100,18 @@ public IMqttClient create() throws Exception {
...
-            String externalEndpointName = AddressSpaceUtils.getExternalEndpointName(addressSpace, "mqtt");
-            mqttEndpoint = kubernetes.getExternalEndpoint(externalEndpointName + "-" + AddressSpaceUtils.getAddressSpaceInfraUuid(addressSpace));
+        if(endpoint == null) {
+            mqttEndpoint = AddressSpaceUtils.getEndpointByServiceName(addressSpace, "mqtt");
+            if (mqttEndpoint == null) {
+                String externalEndpointName = AddressSpaceUtils.getExternalEndpointName(addressSpace, "mqtt");
+                mqttEndpoint = kubernetes.getExternalEndpoint(externalEndpointName + "-" + AddressSpaceUtils.getAddressSpaceInfraUuid(addressSpace));
+            }
+        } else {
+          mqttEndpoint = endpoint;
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/mqtt/MqttClientFactory.java]: Please format code

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=shared-iot

**Pull request merged**